examples/lvgl/Makefile

```diff
@@ -3,6 +3,17 @@
 # Specify this directory relative to the current application.
 TOCK_USERLAND_BASE_DIR = ../..
 
+TOCK_TARGETS := cortex-m0\
+                cortex-m3\
+                cortex-m4\
+                cortex-m7\
+                rv32imac|rv32imac.0x20040060.0x80002800|0x20040060|0x80002800\
+                rv32imac|rv32imac.0x40430060.0x80004000|0x40430060|0x80004000\
+                rv32imac|rv32imac.0x40440060.0x80007000|0x40440060|0x80007000\
+                rv32imac|rv32imac.0x403B0060.0x3FCC0000|0x403B0060|0x3FCC0000\
+                rv32imc|rv32imc.0x20030080.0x10005000|0x20030080|0x10005000\
+                rv32imc|rv32imc.0x41000060.0x42008000|0x41000060|0x42008000
```

Comment:
Why do we need this here? This will very quickly become out of sync wit the main version in Configuration.mk.

Reply:
I added this here as this example cannot be compiled for rv32i. Is there another way of excluding a target?

Reply:
There really shouldn't be any reason why it wouldn't work for `rv32i`. I have seen some weird linking errors before, can you please post the error your experiencing?

Reply:
The issue is the binary is too big (presumably because multiply instructions are being done manually?). I think we should update the linker script to allow for larger apps and remove this special case.

Reply:
I think a rebase and removing this should work.

Reply:
Now I'm getting a different error related to then gcc library.

```
LD        build/rv32i/rv32i.0x00080060.0x40008000.elf
/Users/alexandru/Downloads/riscv64-unknown-elf-toolchain-10.2.0-2020.12.8-x86_64-apple-darwin/bin/../lib/gcc/riscv64-unknown-elf/10.2.0/../../../../riscv64-unknown-elf/bin/ld: cannot find -lstdc++
/Users/alexandru/Downloads/riscv64-unknown-elf-toolchain-10.2.0-2020.12.8-x86_64-apple-darwin/bin/../lib/gcc/riscv64-unknown-elf/10.2.0/../../../../riscv64-unknown-elf/bin/ld: cannot find -lsupc++
collect2: error: ld returned 1 exit status
make: *** [build/rv32i/rv32i.0x00080060.0x40008000.elf] Error 1
```

I am using the 10.2.0 version of gcc for risc-v.

Reply:
That doesn't seem to be an issue in CI, however this error now surfaces: 
```
main.c:65:32: error: 'tocuh_event' undeclared (first use in this function); did you mean 'touch_event'?
   65 |     single_touch_set_callback (tocuh_event, NULL);
      |                                ^~~~~~~~~~~
      |                                touch_event
main.c:65:32: note: each undeclared identifier is reported only once for each function it appears in
```

Reply:
Are you still having an issue with gcc? If so I suspect it is because 2 weeks ago we updated c++ to use gcc 11.2, so your toolchain is out of date.

Reply:
Yes, still fails on my side. For risc-v I am using gcc 10.2.0 from sifive. Do I need to get it from somewhere else? I suppose the gcc (eabi and x86) toolchain installed on my mac is not used for risc-v.